keras_applications/resnet50.py

```diff
@@ -247,6 +248,10 @@ def ResNet50(include_top=True,
             x = layers.GlobalAveragePooling2D()(x)
         elif pooling == 'max':
             x = layers.GlobalMaxPooling2D()(x)
+        else:
+            warnings.warn('You need to be careful about a size of the output. '
```

Comment:
`You need to be careful about a size of the output.` -> `The ResNet50 output shape changed starting with version 2.2.x.`

should there be a way to disable it?

otherwise, LGTM

Reply:
Thank you for the comment, @ahundt. When using `app(weights='imagenet', include_top=False)`, users expect 16x or 32x down-sampled feature maps and all the apps except `ResNet50` return a 32x map (7x7 for a 224x224 input) correctly. The current `ResNet50` results in a 224x down-sampled map (1x1) which is apparently weird. Thus, I think that we need to prevent users from selecting the 224x down-sampled as a feature.

Reply:
I mean "the current ResNet50" is the version before the bottleneck change, as you may know.